.github/workflows/check-plugin.yml

```diff
@@ -26,12 +26,19 @@ jobs:
           echo ::set-env name=CHIME_MESSAGE::$(cat /tmp/chime_message.md)
           exit `cat /tmp/plugin_status.check`
 
-      - name: Send Chime Message
+#      - name: Send Chime Message
+#        if: ${{ always() }}
+#        uses: ros-tooling/action-amazon-chime@master
+#        with:
+#          message: ${{ env.CHIME_MESSAGE }}
+#          webhook: ${{ secrets.CHIME_ODFE_RELEASE }}
+
+      - name: Send Slack Message
         if: ${{ always() }}
-        uses: ros-tooling/action-amazon-chime@master
+        uses: peterzhuamazon/github-slack-action@master
```

Comment:
what's the long term plan for this action?

Reply:
It is just like every single action we grab from the market. Leave it there.
I do not see an issue here.

Reply:
I see. not sure how this action is managed. I think it might be better to mange it at org level. It's ok for me to keep it in personal repo for now.

Reply:
Can it be retrieved from a folder of a repo? If yes we can add it to build repo as a folder

Reply:
@gaiksaya It is a fork from a community member's existing action.
I think it is better to store it in my public repo and getting the upstream updates if possible.

Reply:
What will happen if you go on vacation or leave the company? Will we be able to maintain this?